engine/src/main/scala/cromwell/engine/EngineELF.scala

```diff
@@ -2,21 +2,31 @@ package cromwell.engine
 
 import java.nio.file.{FileSystem, FileSystems}
 
+import com.typesafe.config.ConfigFactory
 import cromwell.backend.wdl.{PureFunctions, ReadLikeFunctions}
 import cromwell.core.WorkflowOptions
+import cromwell.engine.backend.EnhancedWorkflowOptions._
+import cromwell.filesystems.gcs.{GcsFileSystem, GcsFileSystemProvider, GoogleConfiguration}
+import lenthall.config.ScalaConfig._
+import spray.json.JsObject
 import wdl4s.expression.WdlStandardLibraryFunctions
 import wdl4s.values.{WdlFile, WdlValue}
 
 import scala.util.{Failure, Try}
 
 class EngineELF(workflowOptions: WorkflowOptions) extends WdlStandardLibraryFunctions with ReadLikeFunctions with PureFunctions {
+  def gcsFileSystem: Option[GcsFileSystem] = {
+    for {
+      authModeString <- ConfigFactory.load.getStringOption("engine.filesystems.gcs.auth")
+      authMode <- GoogleConfiguration.Instance.auth(authModeString).toOption
```

Comment:
This is why I was complaining about logging the other day... what I really wanted to do was log a warning and then continue on.  My reasoning for semi-silently ignoring this is because somebody might be using only the local backend and the config by default uses `application-default` credentials, but they might not have any GCS stuff setup at all, but that shouldn't necessarily stop the workflow from running.  Up for debate thought, just one idea.